github-sync/github-data/sigstore/users.yaml

```diff
@@ -544,3 +544,8 @@ users:
     teams:
       - name: gitsign-codeowners
         role: member
+  - username: vagarwal
+    role: member
+    teams:
+      - name: triage
```

Comment:
Can we add you to a specific repository, I assume helm-charts? This is triage across all repositories.

Reply:
Yes, just need helm-charts at the moment. Shall I use `teams.name: helm`?

Reply:
Add yourself to this file but make teams `teams: []`, and then add yourself as a collaborator in the helm-charts section like https://github.com/sigstore/community/blob/main/github-sync/github-data/sigstore/repositories.yaml#L714-L715

Reply:
I have made the changes, thank you